test/test.html

```diff
@@ -8,16 +8,8 @@
     <script type="text/javascript">
       mocha.setup('bdd');
     </script>
-    <script src="../src/vendors/cryptojs/components/core-min.js"></script>
-    <script src="../src/vendors/cryptojs/components/sha256-min.js"></script>
-    <script src="../src/vendors/cryptojs/components/enc-base64-min.js"></script>
-    <script src="../src/vendors/cryptojs/rollups/hmac-sha512.js"></script>
-    <script src="../src/vendors/bitcore/browser/bundle.js"></script>
-    <script src="../src/js/consts.js"></script>
-    <script src="../src/js/mnemonic.js"></script>
-    <script src="../src/js/util.bitcore.js"></script>
-    <script src="../src/js/util.bitcoin.js"></script>
-    <script src="../src/js/util.js"></script>
```

Comment:
I guess this hadn't been used for a whole considering it wasn;t working anymore ... double check with someone who knows ...